tree-sitter-stack-graphs/src/loader.rs

```diff
@@ -320,31 +282,17 @@ impl LanguageConfigurationsLoader {
         &mut self,
         path: &Path,
         content: Option<&str>,
-        cancellation_flag: &dyn CancellationFlag,
-    ) -> Result<Option<&mut StackGraphLanguage>, LoadError> {
-        let configuration = match self
+        _cancellation_flag: &dyn CancellationFlag,
```

Comment:
This isn't a trait method impl, and this parameter is not being used, so can we just remove it completely? (Or maybe this is a public API that we don't want to churn?)

Reply:
Good question. I tried to keep the signature analogue to the one of `StackGraphLanguage`. The other reason is that, although we current have longer running implementations for this method, I wouldn't assume these might never exist. Given this is a trait I thought it a bit more future-proof to include it already.

Reply:
Whoops, I cannot read 🤦🏻. You are right, having that parameter here does not make sense. I'll remove it.